### DISCUSSION TITLE
kaiewe

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7774] [general - spritepacks]
can someone let kaiewe know that i emailed him about the spritepacks for monika pls??

**Comment by Ulti70895** [comment]
~Man~ Girl, you created a public issue for something private??? I suggest you be more aware of this. (myopinion)

**Comment by ghost** [comment]
sorry (btw im a girl so pls dont say man)

**Comment by Ulti70895** [comment]
~oh, my bad.

**Comment by ghost** [comment]
its k.
